src/reference/chat/data.json

```diff
@@ -533,7 +649,10 @@
           "user": {
             "user_name": "username",
             "user_id": 102534,
-            "user_roles": ["Pro", "User"]
+            "user_roles": [
+              "Pro",
+              "User"
+            ]
```

Comment:
Can we just pass this file though `jq` or some standard formatter as part of the build process so we don't keep changing it? :D